x/vmbridge/types/protocol.go

```diff
@@ -16,20 +16,29 @@ const (
 
 	SendToEvmSubMsgName = "send-to-evm"
 	EvmCalledMethodName = "mintERC20"
+
+	CallToWasmEventName = "__OKCCallToWasm"
```

Comment:
do not mentioned `okc` in case that it may be support on okbc soon.

Reply:
this line is align with _OKCSendToWasm _OKCSendToIBC